mandatory/1-fix-functions.js

```diff
@@ -10,76 +10,76 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
+function mood(isHappy) {
+
+    if (isHappy) {
+
+        return 'I am not happy'
+    } else {
```

Comment:
Works fine. If line 17 returns, you don't need this `else`, you could just write `return 'I am not happy'` on the next line.